integration-tests/src/test/resources/apps/coherenceapp/gar/pof/Contact.java

```diff
@@ -0,0 +1,129 @@
+<!-- Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
javadoc and comment format is not correct in these files? any reason?
should be 
// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
// Licensed under the Universal Permissive License v 1.0 as shown at
// http://oss.oracle.com/licenses/upl.